### PR TITLE
New bastion ppc64le libvirt sshd 2

### PR DESCRIPTION
**Pull request by hamzy**
This adds a third ppc64le libvirt CI machine to the sshd bastion service

### CODE REVIEW

**Comment by r4f4** [comment]
Perfect. I'll let @deepsm007 review the changes since he's more familiar with that code.

File: clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/deployment.yaml [hunk 1]
```diff
@@ -191,6 +191,101 @@ items:
       port: 443
       targetPort: https-1-3
     type: ClusterIP
+- apiVersion: v1
+  kind: Service
+  metadata:
+    namespace: bastion-ppc64le-libvirt
+    name: sshd-1-0
```
Comment (deepsm007) [inline clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/deployment.yaml:198]: Looking at the changes, you might wanna change the sshd-1-0 -> sshd-2-0 and similar for all of them.

Comment (hamzy) [reply clusters/build-clusters/01_cluster/sshd-bastion/ppc64le-libvirt/deployment.yaml]: Good catch!

**Comment by jaypoulz** [comment]
@hamzy Just a heads up, this patch will interrupt the tunnel since it will kill the currently deployed bastion pod and replace it with a new one with this ports opened. Just for awareness :)

CC @deepsm007 @r4f4

**Comment by hamzy** [comment]
Whee, every day in the CI is a roller coaster! ;)

**Pull request merged**